html/index.html

```diff
@@ -75,9 +74,75 @@ <h2>Software developer</h2>
+      </div>
+      <div class="works_container">
+        <div class="project">
+          <div class="work_img"></div>
+          <h3 class="works_titles">Multi-Post Stories</h3>
+          <p class="project_st">A daily selection of privately personalized reads;no accounts or sign-ups required. has been the industry's 
+             standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+          <div class="work_tools_tags">
+            <div class="tool_tag">css</div>
+            <div class="tool_tag">html</div>
+            <div class="tool_tag">bootstrap</div>
+            <div class="tool_tag">Ruby</div>
+          </div>
+            <button type="button" class="project_btn">See project</button>
+        </div>
+        <div class="project">
+          <div class="work_img"></div>
+          <h3 class="works_titles">Multi-Post Stories</h3>
+          <p class="project_st">A daily selection of privately personalized reads;no accounts or sign-ups required. has been the industry's 
+             standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+          <div class="work_tools_tags">
+            <div class="tool_tag">css</div>
+            <div class="tool_tag">html</div>
+            <div class="tool_tag">bootstrap</div>
+            <div class="tool_tag">Ruby</div>
+          </div>
+            <button type="button" class="project_btn">See project</button>
+        </div>
+        <div class="project"><div class="work_img"></div>
+        <h3 class="works_titles">Multi-Post Stories</h3>
+        <p class="project_st">A daily selection of privately personalized reads;no accounts or sign-ups required. has been the industry's 
+           standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <div class="work_tools_tags">
+          <div class="tool_tag">css</div>
+          <div class="tool_tag">html</div>
+          <div class="tool_tag">bootstrap</div>
+          <div class="tool_tag">Ruby</div>
+        </div>
+          <button type="button" class="project_btn">See project</button>
+        </div>
+        <div class="project">
+          <div class="work_img"></div>
+          <h3 class="works_titles">Multi-Post Stories</h3>
+          <p class="project_st">A daily selection of privately personalized reads;no accounts or sign-ups required. has been the industry's 
+             standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+          <div class="work_tools_tags">
+            <div class="tool_tag">css</div>
+            <div class="tool_tag">html</div>
+            <div class="tool_tag">bootstrap</div>
+            <div class="tool_tag">Ruby</div>
+          </div>
+            <button type="button" class="project_btn">See project</button>
+        </div>
+      </div>
+    </section>
+
```

Comment:
- [ ] Kindly make sure the about section is added to complete this milestone👍

Reply:
hello @rotimiazeez ! thank you for the review, can I have one more hint? I mean, the template doesn't show an about section so I would like to have an idea about how to solve this

Reply:
@Sirvelasque yes you are right 👀 There is no about me section on your template 🤯

![image](https://user-images.githubusercontent.com/15807343/149606923-8477201c-4023-477f-a2d5-b76e07f37723.png)